src/components/explorer/ExplorerSearchSuggestions.tsx

```diff
@@ -1,59 +1,191 @@
 import { PairLogoName } from 'components/common/PairLogoName';
 import { ExplorerSearchProps } from 'components/explorer/ExplorerSearch';
-import { Link, PathNames } from 'libs/routing';
-import { Dispatch, FC, memo, SetStateAction } from 'react';
+import {
+  Dispatch,
+  FC,
+  KeyboardEvent,
+  memo,
+  SetStateAction,
+  useId,
+  useRef,
+  useState,
+} from 'react';
 import { cn } from 'utils/helpers';
+import { ReactComponent as IconClose } from 'assets/icons/times.svg';
+import styles from './explorer.module.css';
```

Comment:
lets not use css modules. This is defeating the biggest advantage of tailwind which is not to have to come up with a bunch of different classnames and it makes it harder to understand the component because I need to switch files to see what styles are applied. It also increases css bundle size.

Use a reusable component instead.

Reply:
As discussed on Telegram, aria-selected is not supported with our current version of Tailwind. We'll keep this part as CSS for now remove it once we update to latest Tailwind version (#774)
The other classes have been moved to a ContainerComponent.